pipeline-modules/deployment-service/aws/istio-app.yaml

```diff
@@ -55,10 +108,234 @@ template: |
+        - |
+          echo "
+          apiVersion: telemetry.istio.io/v1alpha1
+          kind: Telemetry
+          metadata:
+            name: mesh-default
+            namespace: istio-system
+          spec:
+            accessLogging:
+              - providers:
+                - name: envoy " >> enable_istio_access_log.yaml
+          kubectl apply -f enable_istio_access_log.yaml
+        {% else %}  
+        - |
+          echo "
+          apiVersion: telemetry.istio.io/v1alpha1
+          kind: Telemetry
+          metadata:
+            name: mesh-default
+            namespace: istio-system
+          spec:
+            accessLogging:
+              - providers:
+                - name: envoy " >> enable_istio_access_log.yaml
+          #kubectl delete --ignore-not-found=true -f enable_istio_access_log.yaml          
+        {% endif %}
+      {% endif %}  
+
+
+      {% if ingress_install %}
+        - |
+          echo "
+          apiVersion: networking.istio.io/v1alpha3
+          kind: Gateway
+          metadata:
+            name: {{ projectName | lower }}-httpbin-gateway
+          spec:
+            selector:
+              istio: ingressgateway # use Istio default gateway implementation
+            servers:
+            - port:
+                number: 80
+                name: http
+                protocol: HTTP
+              hosts:
+              - '*' "  >> istio_gateway.yaml ;
+          {% if job_type == 'deploy' %}
+          kubectl apply -f istio_gateway.yaml ;
+          {% else %} 
+          #kubectl delete --ignore-not-found=true -f istio_gateway.yaml ;
+          {% endif %}          
+
+        - |
+          echo "          
+          apiVersion: networking.istio.io/v1alpha3
+          kind: VirtualService
+          metadata:
+            name: health
+          spec:
+            gateways:
+            - {{ projectName | lower }}-httpbin-gateway
+            hosts:
+            - '*'
+            http:
+            - match:
+              - uri: 
+                  exact: /healthz/ready
+              route:
+              - destination:
+                  host: istio-ingressgateway.istio-system.svc.cluster.local
+                  port:
+                    number: 15021 " >> istio_healthcheck_vs.yaml ;
+              
+          {% if job_type == 'deploy' %}
+          kubectl apply -f istio_healthcheck_vs.yaml ;
+          {% else %} 
+          #kubectl delete --ignore-not-found=true -f istio_healthcheck_vs.yaml ;
+          {% endif %}
+
+          {% if ingress %}
+          {% for ing in ingress %} 
 
+        - |
+          echo "
+          apiVersion: networking.k8s.io/v1
+          kind: Ingress
+          metadata:
+            name: {{ projectName | lower }}-istio-ingress
+            namespace: istio-system
+            annotations:
+              kubernetes.io/ingress.class: 'alb'
+              alb.ingress.kubernetes.io/scheme: 'internet-facing'
```

Comment:
Should these 2 values be hardcoded?

Reply:
@shwetanshu-cldcvr This not part of helm chart  , this separate manifest file to deploy the ingress on k8s with istio